syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md

```diff
@@ -0,0 +1,573 @@
+
+---
+
+## Execution Cores
+
+Polkadot's Shared Security is powered through the creation and allocation of execution cores.
+
+<img style="width: 1000px;" src="../../assets/img/7-Polkadot/decisions/exotic-scheduling.png" />
+
+Execution cores provide blockspace-as-a-service, and are designed to work with any kind of consensus system.
+
+---
+
+## Trust-Free Interactions
+
+<div class="grid grid-cols-3">
+
+<div>
+
+<img style="height: 500px;" src="../../assets/img/7-Polkadot/xcmp-finalization.svg" />
+
+</div>
+
+<div class="col-span-2 text-left">
+
+A key result of shared security through the Relay Chain is that it keeps track of the state of all parachains and keeps them in lock step.
+
+That means blocks which are finalized on Polkadot imply **finalization of all interactions** between all parachains at the same height.
+
+So, shared security not only secures the individual chains, but the interactions between chains too.
+
+This is continuing to evolve with the addition of "accords" / SPREE.
+
+</div>
+
+</div>
+
+---
+
+## Hybrid Consensus
+
+<div class="grid grid-cols-2">
+
+<div>
+
+### Block Production
+
+<img style="width: 500px;" src="../../assets/img/7-Polkadot/decisions/babe.svg" />
+
+Current implementation is BABE, which is forkful with probabilistic finality.
+
+</div>
+
+<div>
+
+### Finality Gadget
+
+<img style="width: 400px;" src="../../assets/img/7-Polkadot/decisions/grandpa.png" />
+
+Current implementation is GRANDPA, which is robust and scalable against network partitions.
+
+</div>
+
+</div>
+
+---
+
+## Light Client First Mentality
+
+<div class="text-left">
+
+Polkadot has a strong belief that light clients are a necessary component for a Web3 future. It has been uncompromising in enabling first class light client support as a primary part of its development process:
+
+- In-Browser Wasm Client (Substrate Connect)
+  - Wasm state transition function too!
+- Consensus data integrated into block headers
+- Merkle Tries and other LC compatible data structures
+- Maximizing statically known metadata to offset reliance on full nodes.
+
+</div>
+
+---
+
+## On-Chain Runtime & Forkless Upgrades
+
+<div class="grid grid-cols-3">
+
+<div class="col-span-2 text-left">
+
+The Polkadot protocol specification defines a clear separation between the blockchain client and runtime (state transition function).
+
+This is primarily useful to implement the Parachains protocol, but also allows for chains to “forklessly” upgrade their code.
+
+This gives the Polkadot Relay Chain and all connected parachains an evolutionary advantage over others in the blockchain space.
```

Comment:
Forkless upgrades will still be featured for classic substrate parachains but a different approach is being explored for JAM. JAM chain will not be upgrade-able forklessly. Might need to figure out what we want to mention here but it seems it is no longer a type 1 decision